packages/@vue/cli-plugin-unit-jest/presets/typescript/jest-preset.js

```diff
@@ -1,12 +1,19 @@
 const deepmerge = require('deepmerge')
 const defaultPreset = require('../default/jest-preset')
 
+let tsJest = null
+try {
+  tsJest = require('ts-jest')
```

Comment:
Thanks! But this line should be `require.resolve` rather than `require`.
```suggestion
  tsJest = require.resolve('ts-jest')
```